test/test_torchinductor.py

```diff
@@ -1255,6 +1255,30 @@ def fn(x, w, b):
             check_lowp=False,
         )
 
+    def test_conv2d_channels_last(self):
+        m = torch.nn.Sequential(
+            torch.nn.Conv2d(3, 3, 1, 1),
+            ToTuple(),
+        )
+        m = m.to(memory_format=torch.channels_last)
+        x = torch.randn([2, 3, 16, 16]).to(memory_format=torch.channels_last)
```

Comment:
Since the `conv` runs into the channel-last path if the activation or weight is channel-last, please add more test cases to check only the activation or weight is the channels-lat.

Reply:
added.